biocrnpyler/dna_part_rbs.py

```diff
@@ -49,30 +49,19 @@ def update_reactions(self):
         if self.protein is not None:
             reactions += mech_tl.update_reactions(transcript = self.transcript, protein = self.protein, component = self, part_id = self.name)
         return reactions
-    def update_component(self,dna=None,rnas=None,proteins=None,mypos=None):
+    def update_component(self,internal_species=None,**keywords):
```

Comment:
**keywords argument is unused, please remove it

Reply:
We believe this will be needed when we implement global component enumerations.